ee-9/feature-pack/pom.xml

```diff
@@ -428,7 +428,10 @@
                                 <exclude>com.fasterxml.jackson.jaxrs:jackson-jaxrs-base\z</exclude>
                                 <exclude>com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider\z</exclude>
                                 <exclude>com.fasterxml.jackson.module:jackson-module-jaxb-annotations\z</exclude>
-								<!-- end JPMS or ISGi metadata excludes -->
+                                <!-- end JPMS or OSGi metadata excludes -->
+                                <exclude>org.apache.taglibs:taglibs-standard-compat\z</exclude>
+                                <exclude>org.apache.taglibs:taglibs-standard-impl\z</exclude>
+                                <exclude>org.apache.taglibs:taglibs-standard-spec\z</exclude>
```

Comment:
@ropalka This shouldn't be needed. I think the reason you're seeing these still getting transformed is because these 3 are listed as deps in ee-feature-pack/common/pom.xml and and in the ee-feature-pack-common-licenses.xml in that module. But I think having those there is wrong, as they are already included in servlet-feature-pack/ee-8-api. Having them in ee-feature-pack/common as well is duplication.

We don't bring servlet-feature-pack/ee-8-api stuff into this feature-pack so removing it from ee-feature-pack/common should make it disappear from the build of this f-p processing.

If that didn't work for some reason, then instead of adding the above lines the thing to do would be to add excludes to the org.wildfly:wildfly-ee-feature-pack-common dependency element in this pom. We should only use the exclude lines here for things we want incorporated in the f-p but know they don't need to be transformed.

Reply:
Fixed @bstansberry - thanks for the feedback.